Description: Remove cjb.net references
 The domain has been let lapse and is now a holding page.
Author: Olly Betts <olly@survex.com>
Forwarded: no
Last-Update: 2016-01-11

--- antiword-0.37.orig/Docs/ReadMe
+++ antiword-0.37/Docs/ReadMe
@@ -101,7 +101,6 @@ Most recent version
 
 Most recent version of Antiword can be found on the author's website:
 ==>>  http://www.winfield.demon.nl/index.html  <<==
-==>>  http://antiword.cjb.net/  <<==
 
 
 Author
@@ -109,6 +108,5 @@ Author
 
 The author can be reached by e-mail:
 antiword@winfield.demon.nl
-comments@antiword.cjb.net
 
 But PLEASE read the FAQ before you write!!
--- antiword-0.37.orig/Docs/antiword.1
+++ antiword-0.37/Docs/antiword.1
@@ -125,14 +125,8 @@ PostScript output is only available in I
 The most recent released version of Antiword is always available from:
 .br
 http://www.winfield.demon.nl/index.html
-.br
-or try
-.br
-http://antiword.cjb.net/
 .SH AUTHOR
 Adri van Os <antiword@winfield.demon.nl>
-.br
-or try <comments@antiword.cjb.net>
 .sp
 R.F. Smith <rsmith@xs4all.nl> and
 .br
